### ISSUE TITLE
Not working on Android 13 Public Beta 1

### ISSUE THREAD

**Issue by QuackingCanary** [Aliucord/Aliucord#239]
### What happens when the bug or crash occurs?

I just installed Android 13 Beta 1 and I launched Aliucord. But sadly emojis and plugins are broken

### What is the expected behaviour?

Working emojis and plugins

### How do you recreate this bug or crash?

Just launched the app

### Crash-log

```
Replace this text with your crash-log.
```

### Request Agreement

- [x] I did indeed check to make sure the bug or crash report is applicable!

**Comment by Vendicated** [comment]
We can't debug it without more info and logs. Aliucord does work on Android 13 so this is an issue on your end. Join our support channel for help

**Comment by KimihikoAkayasaki** [comment]
It's not like it *just* doesn't work, though.

At first, it looks like Aliucord haven't even injected:
there's no aliu settings in discord settings and
there's no sign that any plugin was loaded.
(It was all ok before updating to A13 Beta 1)

From other things, context menus appear to be not
the ones from before Beta 1, as they would now be
turned to match the system theme (white/dark)

Honestly idk what the reall issue may be,
though it looks like some API's changed again.
(Or some deprecated thing was removed?)

As for logs, as I wrote there's no real way to get them
from inside the app, as it's now looking and working
exactly like the base discord one (maybe except
vanishing emojis... sometimes...) So the question is,
where are the logs system-wise, are they saved somewhere?

Aaand as for the discord server: no one seems to care... (yet)
